recipes-devtools/io4edge-cli/io4edge-cli_1.0.0.bb

```diff
@@ -1,7 +1,10 @@
 inherit go
```

Comment:
Better use the binaries from https://github.com/ci4rail/io4edge-client-go/releases

Reply:
ok, done